src/function/loadEmojione.js

```diff
@@ -111,17 +101,16 @@ function(emojione, uniRegexp, emojioneVersion, readyCallbacks, emojioneSupportMo
 
         emojioneReady(function() {
             var emojiSize = "";
-
             if (options.useInternalCDN) {
-                if(emojioneSupportMode > 4) emojiSize = emojione.emojiSize + "/";
+                if (emojioneSupportMode > 4) emojiSize = emojione.emojiSize + "/";
                 
                 emojione.imagePathPNG = cdn.base + "/png/" + emojiSize;
                 emojione.imagePathSVG = cdn.base + "/svg/" + emojiSize;
                 emojione.imagePathSVGSprites = cdn.base + "/sprites/emojione.sprites.svg";
                 emojione.imageType = options.imageType;
             }
-            if(getSupportMode(emojioneVersion) > 4) {
-                uniRegexp = new RegExp("(" + emojione.regUnicode + ")", "gi");
+            if (getSupportMode(emojioneVersion) > 4) {
+                uniRegexp = emojione.regUnicode;
```

Comment:
I not sure if this will work with images, spans and objects, as I say before, 
`<img src="..." alt="😄"/>` will convert to `<img src="..." alt="<img src="..." alt="😄"/>"/>`

Reply:
But if it really do that, it is a emojione 3 problem, is not?
They have some issues opened in their githutb about the js RegEx.
Are you suggesting that I had should change the emojione default regex? 
I can't keep the same regex of version 2 because of the new emojis available in version 3.

Reply:
emojione don't care about what happens with html code, regex below fixes this

Reply:
it's not emojione problem, but when in emojionearea will pass html images and objects it will convert right in prev version but not now

Reply:
I will take a look at the RegEx and see what can I do.

Reply:
I did some tests and I couldn't reach that scenario you said. I have taken a look at the RegEx and figured out that it beginning is quite identical the changes you have made in emojionearea. 
If you could look at line 160 in https://github.com/emojione/emojione/blob/master/lib/js/emojione.js you can take same conclusion. 

If I have be wrong, can you show me what I can not see?

Reply:
Ok, I see, what's great.